### DISCUSSION TITLE
Suggestions for good GTK themes that work well with i3?

### DISCUSSION THREAD

**Discussion by pupto-bismol** [i3/i3#5461] [Q&A]
Since there are a whole lot to choose from and a lot of GTK themes will randomly bork the look of some windows on i3, are there any good suggestions for what works with i3? Or otherwise a good github source to look through, if it exists? Ideally I'd also have a theme that doesn't look _exactly_ like macOS and integrates pretty well with the flat/square look that is pretty common to i3 and terminal configs.

**Comment by okraits** [comment]
I use the standard Adwaita-dark.

**Comment by orestisfl** [comment]
I've been using Materia-dark-compact for ages
